compile error in `BufferWriter::pipe`

Since the version bump 30 minutes ago I get this error when compiling `env_logger` as secondary dependency in my project
```
error[E0061]: this function takes 3 arguments but 2 arguments were supplied
   --> /home/matthias/.cargo/registry/src/github.com-1ecc6299db9ec823/env_logger-0.9.2/src/fmt/writer/mod.rs:196:43
    |
196 |             WritableTarget::Pipe(pipe) => BufferWriter::pipe(color_choice, pipe),
    |                                           ^^^^^^^^^^^^^^^^^^ ------------ an argument of type `bool` is missing
    |
note: associated function defined here
   --> /home/matthias/.cargo/registry/src/github.com-1ecc6299db9ec823/env_logger-0.9.2/src/fmt/writer/termcolor/shim_impl.rs:26:35
    |
26  |     pub(in crate::fmt::writer) fn pipe(
    |                                   ^^^^
27  |         _is_test: bool,
    |         --------------
28  |         _write_style: WriteStyle,
    |         ------------------------
29  |         pipe: Box<Mutex<dyn io::Write + Send + 'static>>,
    |         ------------------------------------------------
help: provide the argument
    |
196 |             WritableTarget::Pipe(pipe) => BufferWriter::pipe(/* bool */, color_choice, pipe),
    |                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```
May it have been in fact a breaking change?

The crates `inferno` and `bindgen` seem to depend on `env_logger` in my project in case that helps.

--- a/ci/src/task.rs
+++ b/ci/src/task.rs
@@ -57,8 +57,8 @@ pub fn test(args: TestArgs) -> bool {
         command.arg("--lib");
     }
 
-    if let Some(ref features) = features {
-        command.args(["--features", features]);
+    if let Some(features) = &features {
+        command.args(&["--features", features]);
     }
 
     println!("running {:?}", command);

--- a/src/fmt/writer/termcolor/shim_impl.rs
+++ b/src/fmt/writer/termcolor/shim_impl.rs
@@ -24,7 +24,6 @@ impl BufferWriter {
     }
 
     pub(in crate::fmt::writer) fn pipe(
-        _is_test: bool,
         _write_style: WriteStyle,
         pipe: Box<Mutex<dyn io::Write + Send + 'static>>,
     ) -> Self {

